packages/components/src/toolbar-button/accessible-toolbar-button.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { useToolbarItem } from 'reakit/Toolbar';
```

Comment:
What does `useToolbarItem` do?

Reply:
It’s the same as `<ToolbarItem>`, except that it doesn’t return a component. It only returns the HTML props that could be passed to a native `<button>`, for example. That’s how Reakit composes multiple “components” without polluting the React component tree.

More info:
- https://reakit.io/docs/basic-concepts/#props-hooks
- https://reakit.io/docs/composition/#props-hooks

Reply:
More specifically, it adds a new item to the toolbar state (so it knows what’s the previous and next items) and handles key down events (by returning an `onKeyDown` prop) so pressing arrow keys will move focus between items.

Reply:
Yes! Reakit can be used entirely without importing any actual React component. Example: https://codesandbox.io/s/reakit-toolbar-props-hooks-1bxp2